[MWAN] Added new IP ranges for WARP (#14775)
cloudflare/cloudflare-docs: content/magic-wan/zero-trust/warp.md

Use [WARP](/cloudflare-one/connections/connect-devices/warp/) as an {{<glossary-tooltip term_id="on-ramp">}}on-ramp{{</glossary-tooltip>}} to Magic WAN and route traffic from user devices with WARP installed to any network connected with Cloudflare Tunnel or Magic IP-layer tunnels ({{<glossary-tooltip term_id="anycast">}}Anycast{{</glossary-tooltip>}} {{<glossary-tooltip term_id="GRE tunnel" link="/magic-wan/configuration/manually/how-to/configure-tunnels/#add-tunnels">}}GRE{{</glossary-tooltip>}}, {{<glossary-tooltip term_id="IPsec tunnel" link="/magic-wan/configuration/manually/how-to/configure-tunnels/#add-tunnels">}}IPsec{{</glossary-tooltip>}}, or [CNI](/network-interconnect/)). Take advantage of the integration between Magic WAN and [Magic Firewall](/magic-firewall/) and enforce policies at Cloudflare’s global network.

## Prerequisites

Before you can begin using WARP as an on-ramp to Magic WAN, you must set up your [Zero Trust account](/cloudflare-one/setup/#create-a-zero-trust-organization).

## IP ranges

> When connecting a WARP device to Magic WAN, you will have IP addresses from WARP virtual IP address, in the `100.96.0.0/12` range.

Reviewer: ```suggestion
When connecting a WARP device to Magic WAN, you will have virtual IP addresses from WARP, in the `100.96.0.0/12` range.
```

This DNS lookup should return a valid IP address associated with the server or service you are testing for.

Next, test with a browser that you can connect to a service on the WAN by opening a webpage that is only accessible on the WAN. The server can be the same server used in the DNS lookup or another server in the WAN. Connecting using an IP address instead of a domain name should work.

## Double encapsulation

> When a WARP user goes to a location (like an office) with a Magic WAN tunnel already set up, WARP traffic is doubly encapsulated - first by WARP and then by Magic WAN. This is unnecessary, since each onramp method provides full Zero Trust protection.

Reviewer: Nit: We have both `on-ramp` and `onramp` in Dev Docs... Is it a noun vs. adjective thing?

> Since WARP traffic is already protected on its own, Cloudflare recommends that your Magic WAN set up should exclude WARP traffic, and send it to the Internet through regular connections.

Reviewer: ```suggestion
Since WARP traffic is already protected on its own, Cloudflare recommends that you set up Magic WAN to exclude WARP traffic, sending it to the Internet through regular connections.
```
I believe it's a bit more readable this way.
